### DISCUSSION TITLE
Time and date format

### DISCUSSION THREAD

**Discussion by quenthal** [Blackymas/NSPanel_HA_Blueprint#1078] [Ideas]
Local official format for Finnish way of formatting time and date are slightly different that what is available in the blueprint. Is it possible to have the following:

Date format:
weekday D.M (non-capital first letter in weekday, no comma after weekday, no leading zeroes on day or month and dot as separator instead of ':')

Time format:
H.MM (24h, no leading zero on hours, dot seperating hours and minutes).

**Comment by edwardtfn** [comment]
Moved to #1083
